config/account.go

```diff
@@ -29,7 +29,6 @@ type Account struct {
 	LegacyKnownFingerprints []KnownFingerprint `json:"KnownFingerprints,omitempty"`
 	Peers                   []*Peer
 	HideStatusUpdates       bool
-	RequireTor              bool
```

Comment:
How does Golangs JSON parser react to removed fields? Can we actually read existing config files now?